geom/geojson_feature_collection.go

```diff
@@ -153,7 +153,7 @@ func (c *GeoJSONFeatureCollection) UnmarshalJSON(p []byte) error {
 		Type     string           `json:"type"`
 		Features []GeoJSONFeature `json:"features"`
 	}
-	if err := json.Unmarshal(p, &topLevel); err != nil { //nolint:musttag
```

Comment:
`nolint:musttag` was to work around a bug, which was fixed in https://github.com/go-simpler/musttag/issues/93.